geomesa-utils/src/test/scala/org/locationtech/geomesa/utils/bin/BinaryOutputEncoderTest.scala

```diff
@@ -25,20 +25,22 @@ import org.specs2.runner.JUnitRunner
 @RunWith(classOf[JUnitRunner])
 class BinaryOutputEncoderTest extends Specification {
 
-  "BinaryViewerOutputFormat" should {
+  val dateFormat = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
```

Comment:
https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html

replace with this threadsafe version instead for java 8

Reply:
or this: https://commons.apache.org/proper/commons-lang/javadocs/api-3.2/org/apache/commons/lang3/time/FastDateParser.html